### PR TITLE
Add missing pulp-katello to Pulp images

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: containers/roles/pulp/tasks/main.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@
     - pulp-rpm-plugins
     - pulp-puppet-plugins
     - pulp-docker-plugins
+    - pulp-katello
```
Comment (bdunne) [inline containers/roles/pulp/tasks/main.yml:16]: :+1:

File: containers/container.yml [hunk 2]
```diff
@@ -100,7 +100,6 @@ services:
     roles:
       - epel_repositories
       - qpid
-    user: qpid
```
Comment (bdunne) [inline containers/container.yml:103]: Was this change intended for this PR?

Comment (ehelms) [reply containers/container.yml:103]: I can do it separately. Without this change the new qpid image failed to deploy.
